e2e/pages/charts/charts.readonly.spec.js

```diff
@@ -3,8 +3,7 @@ import Hooks from '../../utilities/hooks';
 
 const config = require('../../config/mainConfig');
 
-let hooks; let
-  page;
+let hooks; let page;
```

Comment:
```suggestion
let hooks; 
let page;
```

The reason the linter changed this in the first place is to make the variable declarations their own statements, for the reasons described in https://github.com/airbnb/javascript#variables--one-const. By moving them back to a single line, but as separate statements, this change does support the reasoning behind debugging, and the thing about forgetting a comma, but it doesn't help with the "adding or removing a variable impacts the rest of these variables in the git diff"